rsconf/package_data/rsiviz/nginx.conf.jinja

```diff
@@ -0,0 +1,18 @@
+client_max_body_size 200m;
+
+server {
+    listen {{ nginx.index_port }};
+    root {{ nginx.default_root }};
+    server_name {{ nginx.vhost }};
+    location /{{ rsiviz.url_secret }}/ {
```

Comment:
You can remove this, and instead have a secret service host name. It looks from the proxy_pass that there's no API's except "/"?